site/components/pages/Infinity/GetStartedModal.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import Link from "next/link";
+import Image from "next/image";
+import { Trans } from "@lingui/macro";
+import { urls } from "@klimadao/lib/constants";
+import { Text } from "@klimadao/lib/components";
+import { Modal } from "components/Modal";
+import hiker from "public/hiker.jpg";
+import net from "public/net.jpg";
+import building from "public/building.jpg";
+import ArrowForwardIcon from "@mui/icons-material/ArrowForward";
+import * as styles from "./styles";
+
+export interface GetStardedModalProps {
+  showModal: boolean;
+  setShowModal: (value: boolean) => void;
+}
+
+const GetStartedModal = (props: GetStardedModalProps) => {
+  return (
+    <Modal
+      onToggleModal={() => props.setShowModal(false)}
+      showModal={props.showModal}
+      closeOnBackgroundClick={true}
+      title="Get Started"
+    >
+      <div className={styles.modalContainer}>
+        <Link href={urls.creolBusinessCalculator} passHref>
```

Comment:
The child of `<Link>` should always be an `<a />` tag so it can be tab focused. Also helps for a11y and screen readers.
Then you don't need `passHref`

Reply:
updated